doc/user/methods/adaboost_regressor.md

```diff
@@ -0,0 +1,582 @@
+## `AdaBoostRegressor`
+
+The `AdaBoostRegressor` class implements the 'adaptive boosting' regressor 
+AdaBoost.R2.
+An AdaBoost regressor is a meta-estimator that begins by fitting a regressor 
+on the original dataset and then fits additional copies of the regressor on the
+same dataset but where the weights of instances are adjusted according to the 
+error of the current prediction. As such, subsequent regressors focus more on 
+difficult cases.
+
+`AdaBoostRegressor` is useful for regressions; i.e., predicting 
+_continuous values_ (`0.2`, `1.3`, etc.). For predicting _discrete label_
+(classification), see [`AdaBoost`](adaboost.md)
+
+mlpack's `AdaBoostRegressor` class offers configurability via template
+parameters.
+
+#### Simple usage example:
+
+```c++
+//Train a adaboost regressor on random numeric data and make prediction.
+
+//All data and responses are uniform random; this uses 10 dimensional data.
+//Replace with a data::Load() call or similar for a real application.
+arma::mat dataset(10, 1000, arma::fill::randu); // 1000 points.
+arma::rowvec responses = arma::randn<arma::rowvec>(1000);
+arma::mat testDataset(10, 500, arma::fill::randu); // 500 test points.
+
+mlpack::AdaBoostRegressor<> abr;     // Step 1: create model.
+abr.Train(dataset, responses);         // Step 2: train model
+arma::rowvec predictions;
+abr.Predict(testDataset, predictions); // Step 3: use model to predict.
+
+// Print some information about the test predictions.
+std::cout << arma::accu(predictions > 0.7) << " test points predicted to have"
+    << " responses greater than 0.7." << std::endl;
+std::cout << arma::accu(predictions < 0) << " test points predicted to have "
+    << "negative responses." << std::endl;
+```
+<p style="text-align: center; font-size: 85%"><a href="#simple-examples">More examples...</a></p>
+
+#### Quick links:
+
+ * [Constructors](#constructors): create `AdaBoostRegressor` objects.
+ * [`Train()`](#training): train model.
+ * [`Predict()`](#prediction): predict values with a trained model.
+ * [Other functionality](#other-functionality) for loading, saving, and
+   inspecting.
+ * [Examples](#simple-examples) of simple usage and links to detailed example
+   projects.
+ * [Template parameters](#advanced-functionality-template-parameters) for custom
+   behavior.
+
+#### See also:
+
+ * [`DecisionTree`](decision_tree.md)
+ * [`DecisionTreeRegressor`](decision_tree_regressor.md)
+ * [AdaBoost](adaboost.md)
+ * [AdaBoost on Wikipedia](https://en.wikipedia.org/wiki/AdaBoost)
+ * [Decision tree on Wikipedia](https://en.wikipedia.org/wiki/Decision_tree)
+ * [Improving Regressors using Boosting Techniques](https://dl.acm.org/doi/10.5555/645526.657132)
```

Comment:
Can we link directly to a PDF?  Many users will not be able to access the text directly from ACM.